__tests__/secrets.test.js

```diff
@@ -0,0 +1,40 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const UserService = require('../lib/services/UserService');
+
+const mockUser = {
+  email: 'niki@hite.com',
+  password: 'august3'
+};
+
+const registerAndLogin = async (userProps = {}) => {
+  const password = userProps.password ?? mockUser.password;
+  
+  const agent = request.agent(app);
+  const user = await UserService.create({ ...mockUser, ...userProps });
+  console.log(user);
+  const { email } = user;
+  await agent.post('/api/v1/users/sessions').send({ email, password });
+  return [agent, user];
+};
+describe('backend-express-template routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+  it('/get secrets should return secrets for authenticated users', async () => {
+    // const [agent] = await registerAndLogin();
+    const agent = request.agent(app);
+    await agent.post('/api/v1/users').send(mockUser);
+    let res = await agent
```

Comment:
great use of the let declaration here, since you're changing the definition of the variable on line 33!